scripts/hashauthservice

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import csv
+from functools import wraps
+import os
+from urllib.parse import urlparse
+
+import requests
+from traitlets.config import PyFileConfigLoader
+
+from tornado.ioloop import IOLoop
```

Comment:
I hope this is listed in the dependencies somewhere?

Reply:
Not explicitly, for two reasons:
1. Tornado is a dependency of JupyterHub, so if you've got JH installed, you'll be okay here.
2. I feel like this service is a little bonus, but the meat of this project is the authenticator itself.  If you just want to use the authenticator, you shouldn't have to install the things to make the service work.  I don't know how to represent this in the setup.py file, so I haven't put anything additional there.